### ISSUE TITLE
Print pdf downloads an invalid pdf file - ANDROID 

### ISSUE THREAD

**Issue by martinbedouret** [cboard-org/cboard#1657]
this is valid just for Android app. The fix #1649 worked well on the Web version, but it downloads an invalid file for Android. It works fine using the export pdf from the settings.

**Comment by RodriSanchez1** [comment]
I couldn't reproduce this issue. @martinbedouret, could you provide more details on the steps to reproduce it?

**Comment by martinbedouret** [comment]
you are right @RodriSanchez1 , I have tested it again and can't reproduce anymore. Closing.
